eq-author/src/components/withEntityEditor/index.js

```diff
@@ -92,7 +92,7 @@ const withEntityEditor = entityPropName => WrappedComponent => {
       this.props.onSubmit(this.entity);
     };
 
-    render() {
+    render = () => {
```

Comment:
This change isn't needed. Its not passed anywhere so does not need to be bound explicitly

Reply:
Accidentally left over from previous changes. Now reverted.